pkg/apis/admin/v1beta2/doc.go

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2018, EnMasse authors.
```

Comment:
Update year (applies to other files as well)

Reply:
Fixed.